### PR TITLE
Create groom me file for groom changelog branch

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Changelog for 1.20.0-beta.1
# Why

Shows best Changelog edits for new beta

# How Tested

Changelog show in diff

### CODE REVIEW

File: GROOMME.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Please groom the changelog and then delete me.
```
Comment (JoshuaLai) [inline GROOMME.md:1]: Can we delete this now that is has been groomed?

**Review by JoshuaLai — APPROVED** [review]

File: GROOMME.md (hunk 1 above)
Comment (dmceachernmsft) [reply GROOMME.md:1]: yes forgot that step will delete

**Review by palatter — APPROVED** [review]

File: packages/communication-react/CHANGELOG.beta.md [hunk 2]
```diff
@@ -1,40 +1,30 @@
 # Change Log - @azure/communication-react
 
-<!-- This log was last generated on Tue, 01 Oct 2024 17:03:44 GMT and should not be manually modified. -->
-
-<!-- Start content -->
-
 ## [1.20.0-beta.1](https://github.com/azure/communication-ui-library/tree/1.20.0-beta.1)
 
 Tue, 01 Oct 2024 17:03:44 GMT 
 [Compare changes](https://github.com/azure/communication-ui-library/compare/1.19.0...1.20.0-beta.1)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.beta.md:6]: Should we be comparing against 1.19.0 or 1.19.0-beta.3?

File: packages/communication-react/CHANGELOG.beta.md [hunk 3]
```diff
@@ -1,40 +1,30 @@
...
-- Adding logic to flag when the adapter is in the middle of creation ([PR #5182](https://github.com/azure/communication-ui-library/pull/5182) by 9044372+JoshuaLai@users.noreply.github.com)
-- Add new sub package to communication react to import the loader functions ([PR #5106](https://github.com/azure/communication-ui-library/pull/5106) by dmceachern@microsoft.com)
-- Fix Call Controls to allow proper customization of buttons shown ([PR #5193](https://github.com/azure/communication-ui-library/pull/5193) by dmceachern@microsoft.com)
+- Introduce logic to flag when the `AzureCommunicationServicesChatAdapter` is in the middle of creation when using the react hook: `useAzureCommunicationServicesChatAdapter` ([PR #5182](https://github.com/azure/communication-ui-library/pull/5182) by 9044372+JoshuaLai@users.noreply.github.com)
+- Fix call control configuration API. There was an issue where some controls weren't hidden when configured to be removed. ([PR #5193](https://github.com/azure/communication-ui-library/pull/5193) by dmceachern@microsoft.com)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.beta.md:27]: mega nit: remove period

**Pull request merged**